### PR TITLE
Remove activity pages feature flags from FEATURES_PENDING_REMOVAL

### PR DESCRIPTION
**Pull request by robertknight**
The removal of the feature flags has now been deployed all the way out to production as per the comments above the `FEATURES_PENDING_REMOVAL` variable.

### CODE REVIEW

**Comment by robertknight** [comment]
I just noticed this locally:

```
12:41:52 web    |     reraise(type(exception), exception, tb=exc_tb, cause=cause)
12:41:52 web    |   File "/home/robert/.virtualenvs/h/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1182, in _execute_context
12:41:52 web    |     context)
12:41:52 web    |   File "/home/robert/.virtualenvs/h/local/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 469, in do_execute
12:41:52 web    |     cursor.execute(statement, parameters)
12:41:52 web    | IntegrityError: (psycopg2.IntegrityError) update or delete on table "feature" violates foreign key constraint "fk__featurecohort_feature__feature_id__feature" on table "featurecohort_feature"
12:41:52 web    | DETAIL:  Key (id)=(37) is still referenced from table "featurecohort_feature".
12:41:52 web    |  [SQL: 'DELETE FROM feature WHERE feature.name NOT IN (%(name_1)s, %(name_2)s, %(name_3)s)'] [parameters: {'name_2': u'orphans_tab', 'name_3': u'total_shared_annotations', 'name_1': u'defer_realtime_updates'}]
```

We'll need to make sure no feature cohorts are referencing the flag before deploying to production, or add code to remove any entries from the `featurecohort_feature` table that still reference remove flags.

**Comment by robertknight** [comment]
Slack discussion: https://hypothes-is.slack.com/archives/backend/p1484052480001639

TL;DR - Add "ON DELETE" triggers to remove feature flags from existing cohorts that have them enabled when the flag is actually removed.

**Comment by chdorner** [comment]
@robertknight what's the status on this?

If adding this `ON DELETE` trigger is a considerable amount of work and testing (which it probably is), then we should maybe create a ticket in the backlog and close this PR for the time being?

**Comment by robertknight** [comment]
I'm not going to get to this today. Closing for the moment until the `ON DELETE` triggers have been implemented.
